_emails/kalamazooMi.md

```diff
@@ -26,32 +26,29 @@ recipients:
-  and needs to change. Kalamazoo needs dollars for services that actually improve
-  and lift up our community. Dollars for education, for food assistance, for housing,
-  for people. Not policing.\n\nSincerely,\n\n[YOUR NAME]\n\n[YOUR ADDRESS]\n\n[YOUR
-  EMAIL]\n\n[YOUR PHONE NUMBER]"
+body: Hi, my name is [YOUR NAME] and I live at [YOUR ADDRESS, YOUR CITY, YOUR COUNTY]. I’m writing to let you know that I think that Kalamazoo Department of Public Safety’s use of chemical agents on peaceful protesters in late May and early June, including youth, constituted excessive force. I also think that the curfew and the presence of the National Guard in our city on June 2 did more to harm and scare us than it did to keep us safe.
+
+This kind of behavior is a long standing pattern with KDPS, a police force with multiple questionable in custody deaths, excessive force accusations, a documented habit of pulling over predominantly black drivers, and enough unnecessary lethal equipment to outfit a small army. The time for hiding behind canned apologies, promises of diversity training and “community policing” is over. It’s time to reduce police spending and reallocate those funds to programs that have real tangible benefits, especially to communities harmed by decades of over-policing and discrimination.
+
+Kalamazoo Public Safety takes up 48% of the entire general fund budget for 2020. Other public services account for a paltry 8% and parks and recreation a depressing 4% of budget. That is outrageous and needs to change. Kalamazoo needs dollars for services that actually improve and lift up our community. Dollars for education, for food assistance, for housing, for people. Not policing.
+
+Additionally, I want to see the following actions taken by KDPS and the City Commission to hold them accountable for their violence toward our community:
+IMMEDIATELY:
+KDPS will cease the use of tear gas or other chemical weapons
+KDPS will cease the use of riot gear or other provocative dress when interacting with the public
+KDPS will wear body cams at all times, always on and unobstructed
+The Kalamazoo City Commission will order an independent investigation into KDPS response to public protests beginning May 30
+KDPS will provide a weekly public briefing on the progress of their own investigations into police conduct related to public protests beginning May 30
+WITHIN 2 WEEKS:
+KDPS will develop and ongoing plan for quarterly reporting, to the public, of all complaints against the department or its officers, and the resolution of each
+WITHIN 3 MONTHS:
+ALL KDPS officers who interact with the public will have participated in racial healing circle with members of the community, led by trained facilitators
+KDPS will adopt a policy requiring its officers live within the boundaries of the area the department serves
+
+Please keep my interests in mind as you continue to discuss these matters and work toward justice in our community.
+
+Sincerely,
+
+[YOUR NAME]
 ---
```

Comment:
This formatting won't actually build correctly.

I would use this one as a reference: https://github.com/teddywilson/defund12.org/pull/178/files

There should be a pipe character after body: |-

and then everything below it should be indented.

As a second note, maybe we should create a new line in between some of these sections?

WITHIN 2 WEEKS:
......

WITHIN 3 MONTHS:
....